lib/galaxy/managers/users.py

```diff
@@ -271,14 +271,14 @@ def by_api_key(self, api_key, sa_session=None):
             raise exceptions.AuthenticationFailed("Provided API key has expired.")
         return provided_key.user
 
-    def check_master_api_key(self, api_key):
-        master_api_key = getattr(self.app.config, "master_api_key", None)
-        if not master_api_key:
+    def check_bootstrap_admin_api_key(self, api_key):
+        bootstrap_admin_api_key = getattr(self.app.config, "bootstrap_admin_api_key", None)
+        if not bootstrap_admin_api_key:
```

Comment:
```suggestion
        if not self.app.config.bootstrap_admin_api_key:
```
The attribute is created automatically, right?